go.mod

```diff
@@ -22,16 +22,11 @@ require (
 	github.com/spf13/afero v1.4.1
 	github.com/spf13/viper v1.7.1
 	k8s.io/api v0.18.9
+	k8s.io/apiextensions-apiserver v0.18.9 // indirect
 	k8s.io/apimachinery v0.18.9
 	k8s.io/client-go v0.18.9
+	sigs.k8s.io/cluster-api v0.3.10 // indirect
 	sigs.k8s.io/yaml v1.2.0
 )
 
-replace (
-	github.com/ghodss/yaml => github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/lib/pq => github.com/lib/pq v1.3.0
-	github.com/mailru/easyjson => github.com/mailru/easyjson v0.7.0
-	github.com/mattn/go-colorable => github.com/mattn/go-colorable v0.1.2
-	github.com/mattn/go-isatty => github.com/mattn/go-isatty v0.0.9
-	github.com/mattn/go-runewidth => github.com/mattn/go-runewidth v0.0.4
-)
+replace sigs.k8s.io/cluster-api => github.com/giantswarm/cluster-api v0.3.10-gs
```

Comment:
I created these replaces but not sure why they got so complicated.

Setting this for our fork as per this news-dev post. https://gigantic.slack.com/archives/C04TGHDEF/p1602251714027300